### PR TITLE
update console stub return types and docblocks

### PR DESCRIPTION
**Pull request by browner12**
- add `void` return type
- remove unnecessary docblocks

syncs with a [docs change](https://github.com/laravel-zero/docs/commit/c402dd2ff5828361c4ef58f0ecede44923905bcb)

### CODE REVIEW

File: src/Commands/stubs/console.stub [hunk 1]
```diff
@@ -23,19 +23,14 @@ class DummyClass extends Command
 
     /**
      * Execute the console command.
-     *
-     * @return mixed
      */
-    public function handle()
+    public function handle(): void
```
Comment (owenvoke) [inline src/Commands/stubs/console.stub:27]: I don't think we should add the typehint here for now.

**Review by owenvoke — CHANGES_REQUESTED** [review]
I was thinking of merging this, however I remembered the issues which people brought up when Laravel added return types to these stubs.

Related, I do think we should update the stubs within Laravel Zero to use the `{{ class }}` format that Laravel uses (rather than `DummyClass`, etc.)

File: src/Commands/stubs/console.stub (hunk 1 above)
Comment (browner12) [reply src/Commands/stubs/console.stub:27]: I'm okay with this, as `laravel/framework` also does not define the `void` return type in the console `handle()` method, but they do omit the return type. I would suggest we update the docs then too, to match this.

**Comment by browner12** [comment]
should we also update the "DummyNameSpace"?

**Comment by browner12** [comment]
okay, everything should be synced up now.

I tried this stub on a local Laravel Zero app I have and the file generated correctly.

Should be good to review.

**Review by owenvoke — APPROVED** [review]

**Pull request merged**